synfig-studio/src/gui/docks/dockable.cpp

```diff
@@ -156,6 +161,30 @@ Dockable::attach_dnd_to(Gtk::Widget& widget)
 	widget.signal_drag_data_received().connect( sigc::mem_fun(*this, &Dockable::on_drag_data_received ));
 }
 
+void Dockable::dettach()
```

Comment:
Maybe better to rename it to `Undock` instead of `Detach`?

Reply:
I chose `dettach` because the class has `attach_dnd_to()` . And now I saw made a typo XD
But I can change it to `undock` without any complaining.

Reply:
Ok, when I think better to rename method to `detach` (to save attach/detach naming) and `Gtk::MenuItem(_("Undock panel")`

Reply:
Done